commands/create.go

```diff
@@ -128,7 +128,13 @@ func create(command *Command, args *Args) {
 	localRepo, err := github.LocalRepo()
 	utils.Check(err)
 
-	originName := "origin"
+        var originName string
+        if flagOriginRemoteName := args.Flag.Value("--remote-name"); flagOriginRemoteName != "" {
```

Comment:
This looks good so far. You also have to add the `--remote-name` flag to the flags documentation near the start this file for the flag to start working. Something like:

```
	--remote-name <REMOTE>
		Set the name for the new git remote (default: "origin").
```

The Go code that you wrote here looks great, but you can also write it simpler:

```go
originName := args.Flag.Value("--remote-name")
if originName == "" {
  originName = "origin"
}
```